pubpub/platform
# Render multivalue fields

## Issue(s) Resolved

External form side of https://github.com/pubpub/platform/issues/613 and https://github.com/pubpub/platform/issues/612. Will do select dropdown in a separate PR, but should look pretty similar.

## High-level Explanation of PR
Can now fill out numeric and string array fields on the fill page of a form.

## Test Plan

* Add a numeric array and a string array field
* Add these to a form, one as a checkbox group and one as a radio group
* Open the form fill page. The checkbox and radio group elements should render as expected
  * In the form builder, try adding the 'other' field and play around with it on the fill page
  * Try playing with the "User should select" and "Checkboxes" options and make sure the validation works on the fill form

## Screenshots (if applicable)


https://github.com/user-attachments/assets/20df6f98-d6e5-460c-9791-fabf1e2a42cb



## Notes
